security/s2a/internal/crypter/testutil/common.go

```diff
@@ -0,0 +1,55 @@
+package testutil
+
+import "encoding/hex"
+
+// Constants indicating whether the test vector is valid or not.
+const (
+	ValidResult   = "valid"
+	InvalidResult = "invalid"
+)
+
+// CryptoTestVector is a struct representing a test vector for an S2AAeadCrypter
+// instance.
+type CryptoTestVector struct {
+	Id                                          int
+	Key, Plaintext, Ciphertext, Tag, Nonce, Aad []byte
+	Result, Comment                             string
+	AllocateDst                                 bool
+}
+
+// TestVector is a struct for a WycheProof test vector.
+type TestVector struct {
+	TcId    int    `json:"tcId"`
```

Comment:
Please call this `TcID`:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done